oauth2_http/java/com/google/auth/oauth2/UserAuthorizer.java

```diff
@@ -164,6 +165,20 @@ public TokenStore getTokenStore() {
    * @return The URL that can be navigated or redirected to.
    */
   public URL getAuthorizationUrl(String userId, String state, URI baseUri) {
+    return this.getAuthorizationUrl(userId, state, baseUri, null);
+  }
+
+  /**
+   * Return an URL that performs the authorization consent prompt web UI.
+   *
+   * @param userId Application's identifier for the end user.
+   * @param state State that is passed on to the OAuth2 callback URI after the consent.
+   * @param baseUri The URI to resolve the OAuth2 callback URI relative to.
+   * @param additionalParameters Additional query parameters to be added to the authorization url
```

Comment:
nit: missing ending period; url => URL to be consistent.
Rest of it looks good.